src/v0/destinations/intercom/transform.js

```diff
@@ -27,8 +27,10 @@ function getCompanyAttribute(company) {
       // the key is not in ReservedCompanyProperties
       if (!ReservedCompanyProperties.includes(key)) {
         const val = company[key];
-        if (val) {
+        if (!Array.isArray(val) && typeof val !== 'object') {
```

Comment:
https://www.geeksforgeeks.org/how-to-check-if-the-value-is-primitive-or-not-in-javascript/